.circleci/config.yml

```diff
@@ -29,7 +29,26 @@ jobs:
           # don't limit this to the number of allocated cores, the job is
           # likely to get OOMed and killed.
           GOOPTS: "-p 2"
-  publish:
+  publish_infra:
+    executor: golang
+    steps:
+    - checkout
+    - setup_remote_docker:
+        version: 18.06.0-ce
+    - run:
+        name: Setup environment variables
+        command: |
+          echo "export DOCKER_IMAGE_TAG=master" >> $BASH_ENV
+          echo "export DOCKER_ORG=${DOCKER_ORG:-prominfra}" >> $BASH_ENV
+    - prometheus/publish_images:
+        container_image_name: infra
+        dockerfile_path: "infra/Dockerfile"
+        dockerbuild_context: "infra/"
```

Comment:
don't understand why you still need context? if promu builds the binary inside the tool folder and the docker file is there can just run docker build within that folder.

Reply:
the prometheus circle ci orb runs directly from repository root: https://github.com/prometheus/circleci/blob/c4c61ad3812f4e83b0d8661dc5dce0556e1fda05/src/commands/publish_images.yml#L44

if we want to build from the directory directly then we have to `cd` into the directory anyway. eg. `cd infra/` in this case, and do that for every other tool.

Reply:
got it, so you are saying it is more simple this way

Reply:
um currently it's less work, because the orb already has the build and publish commands and the `/Makefile` has docker targets that the circleci orb uses. If we `cd` then we'll probably have to change a couple of things. Maybe have a script/make target to cd and build and push but I think I tried doing a variant of that in a PR previously but then we decided to use the orb.

Reply:
ok not a problem , lets move forward.